README.md

```diff
@@ -420,6 +426,47 @@ Style: 0 / 0
 Total: 100 / 100
 ```
 
+##### Resubmitting a Student's Code by ID
+
+Proxy resubmit a student's code by their previous submission ID by using the `autograder.cli.courses.assignments.submissions.proxy.resubmit` command.
+By default, proxy resubmit targets the student's most recent submission:
+```sh
+python3 -m autograder.cli.courses.assignments.submissions.proxy.resubmit --proxy-email student@test.edulinq.org
+```
+
+The output may look like:
+```
+Autograder transcript for assignment: HO0.
+Grading started at 2025-04-02 12:45 and ended at 2025-04-02 12:45.
+Task 1.A (my_function): 40 / 40
+Task 2.A (test_my_function_value): 30 / 30
+Task 2.B (TestMyFunction): 30 / 30
+Style: 0 / 0
+   Style is clean!
+
+Total: 100 / 100
+```
+
+Finally, we can target a specific submission ID through `target-submission`.
```

Comment:
s/Finally, we/We/

I assume this is just a carryover from when there were three examples.